ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -322,11 +322,12 @@ public void replicaThreadTest() throws Exception {
 
```

Comment:
need to chat about whether this guarantees coverage of all the cases we want it to (if it does not, lets make a separate PR for that since this is unrelated to metrics)

Reply:
sure. lets take it outside this patch.